src/cluster/Cargo.toml

```diff
@@ -34,21 +37,22 @@ serde = "1.0.115"
 serde_json = "1.0.57"
 thiserror = "1.0.20"
 async-trait = "0.1.21"
+color-eyre = { version = "0.5.8", default-features = false, optional = true }
 structopt = { version = "0.3.16", default-features = false, optional = true }
 prettytable-rs = { version = "0.8.0", optional = true }
 
 # Fluvio dependencies
-fluvio-helm = { version = "0.1.0", git = "https://github.com/infinyon/fluvio-helm", branch = "main" }
 fluvio = { version = "0.2.3", path = "../client", default-features = false }
-fluvio-controlplane-metadata = { version = "0.2.0", path = "../controlplane-metadata", features = ["k8"] }
+fluvio-helm = { git = "https://github.com/infinyon/fluvio-helm", rev = "6e15fd2c789c33ba8bae51a369a0983e177c2f78" }
```

Comment:
since helm changes are merged into main, can remove `rev`

Reply:
I could see it either way here, but I've heard that it's nice to use `rev` when using git dependencies because it will give you a more reproducible build. Using `branch = main` is kind of like using cargo where you specify `version = "*"` because any change upstream will automatically be pulled in. If somebody makes breaking changes on the main branch, this would immediately fail the build here, but pinning a `rev` version would require us to opt into that change and fix things that break. If you still want me to change it I will, that was just my reason for doing it this way.

Reply:
how if I publish helm repo as is?

Reply:
I think that would be fine, I don't know of any other changes that would need to go there

Reply:
`version 0.1.0` is published